manifests/concourse-manifest/extensions/datadog.yml

```diff
@@ -1,3 +1,9 @@
+releases:
+  - name: riemann
+    url: https://s3-eu-west-1.amazonaws.com/michael-state/riemann-2%2Bdev.1.tgz
```

Comment:
Good point. Given that this is not a fork and we don't have our final bucket created yet (and pipeline to do automated builds in place), I'll create a release here on paas-cf and add the tarball there.